#17. Can one block of except statements handle multiple exception? 

# Yes, a single except block can handle multiple exceptions in Python. You can do this by specifying a tuple of exception types in the 
# except clause. If any of the exceptions listed in the tuple are raised, the corresponding except block will be executed.